cmd/dotmesh-server/rpc.go

```diff
@@ -2008,7 +1999,7 @@ func (d *DotmeshRPC) AddCollaborator(
 
 	user := auth.GetUser(r)
 
-	authorized, err := crappyTlf.AuthorizeOwner(user)
+	authorized, err := d.usersManager.Authorize(user, false, &crappyTlf)
```

Comment:
You kept the name `crappyTlf` ;-)

Reply:
Who am I to argue? ;-)

...actually, I've no idea why it's called that!